### ISSUE TITLE
Need to do workaround "block" and "allow" of camera and micro every time

### ISSUE THREAD

**Issue by jknphy** [jangouts/jangouts#218]
In Chromium I always have to go "always block" and "always allow" before any conversation or even after some other users reconnect.
In Firefox is not working at all, camera is not working and some people can hear me and others not.
I would like to focus on solution in Chromium.

**Comment by dgdavid** [comment]
> I would like to focus on solution in Chromium.

@jknphy, please don't do it :(

https://twitter.com/jyasskin/status/935376631681597441

**Comment by jknphy** [comment]
I agree :) but in Firefox works in so random way that I am not sure how to provide more info. Is there any troubleshooting guide?

**Comment by ancorgs** [comment]
@jknphy I guess you are talking about the SUSE internal instance. Do you connect to it through VPN? Could you do some tests in the external demo instance at https://jangouts.tk/callstats ?

**Comment by jknphy** [comment]
@ancorgs  I am not using VPN, I just did some tests in this demo instance.

**Comment by ancorgs** [comment]
Could you re-check at this new demo site that executes version 0.6.0? https://li2023-182.members.linode.com/
